src/topology/bounded_continuous_function.lean

```diff
@@ -399,44 +425,17 @@ instance : add_comm_group (α →ᵇ β) :=
   ..bounded_continuous_function.has_neg,
   ..bounded_continuous_function.has_zero }
 
-@[simp] lemma coe_diff : (f - g) x = f x - g x := rfl
+@[simp] lemma sub_apply : (f - g) x = f x - g x := rfl
```

Comment:
This is a good change. You could also maybe register `coe_sub`, i.e., `(f - g : alpha -> beta) = (\lambda x, f x - g x)` and put the simp attribute instead on this one. Same thing for `coe_add` and `coe_neg` above.

Reply:
May be we can use `coe_sub : (f - g : α → β) = has_sub.sub`, then it will be usable both for left-to-right and right-to-left rewrites.

Reply:
@urkud That's a smart trick (although not very readable, but I think I don't care). We should maybe use it more often.

Reply:
You mean
```
lemma coe_sub : ((f - g : α →ᵇ β) : α → β) = f - g
```
Yes, it's a good formulation.

Reply:
I was wrong, there is no easy way to avoid `*_apply` lemmas if we want to rewrite right to left.

Reply:
Done.